build.gradle

```diff
@@ -384,11 +383,12 @@ project('spring-rabbit') {
 
 		api project(':spring-amqp')
 		api "com.rabbitmq:amqp-client:$rabbitmqVersion"
-		optionalApi "com.rabbitmq:http-client:$rabbitmqHttpClientVersion"
 		optionalApi 'org.springframework:spring-aop'
 		api 'org.springframework:spring-context'
 		api 'org.springframework:spring-messaging'
 		api 'org.springframework:spring-tx'
+		optionalApi 'org.springframework:spring-web'
```

Comment:
This one comes as a transitive dep from WebFlux: `api(project(":spring-web"))`.
So, why to use it explicitly?

Reply:
Not sure; there was some class in it that wasn't on the CP until I added this; will try to remove.